### ISSUE TITLE
put-subscription --es-query bug

### ISSUE THREAD

**Issue by amarjandu** [HumanCellAtlas/data-store#2248]
upon running `hca dss put-subscription --es-query '{}' --replica aws --callback-url https://www.example.com --hmac-key-id 96 --hmac-secret-key 2333` the subscription makes its way into the v2-sub-dynamodb table , rather than Elasticsearch.  maybe related to the empty query parameters

**Comment by kislyuk** [comment]
Not sure how this is a bug - the default should be v2 subscriptions, as v1 will be deprecated eventually, right?

**Comment by xbrianh** [comment]
@kislyuk Yes, v2 subscriptions are the default, and v1 subscriptions are slated for depreciation.

However, for the case Amar mentions the user has passed in an explicit ES query by specifying the parameter `--es-query '{}'`. This should probably produce a v1 subscription.

**Comment by kislyuk** [comment]
Ah great point, I didn't notice that.
